pulsar-broker/src/test/resources/authentication/basic/basic.json

```diff
@@ -0,0 +1,6 @@
+{
+  "super": {
+    "password": "superPassword",
```

Comment:
Seriously? You should google "htpasswd".

Reply:
Agree, let's use common format for basic user:password file. That would eliminate the needs for a specific script/tool to manage it.

Reply:
Fixed it and made userId equivalent to role.